### PR TITLE
EI-1310 Update for breaking change

### PR DESCRIPTION
**Pull request by captnz**
This is for breaking changes in Iotics api
https://ioticlabs.atlassian.net/wiki/spaces/DT/pages/1001750529/PeerId+-+HostTwinDID

I will create another PR for releasing when this has been reviewed

See ticket for how to test against a local host

### CODE REVIEW

File: src/feeds.ts [hunk 1]
```diff
@@ -114,15 +110,13 @@ export const shareFeedDataApi = async (
...
         request.setHeaders(headers as unknown as pbCommonModel.Headers);
-        const twinID = new pbCommonModel.TwinID();
-        twinID.setValue(twinId);
-        const feedID = new pbCommonModel.FeedID();
-        feedID.setValue(feedId);
-        const feed = new pbFeedModel.Feed();
-        feed.setTwinid(twinID);
-        feed.setId(feedID);
+
+        const feedID = new pbFeedModel.FeedID();
```
Comment (zoltaniotic) [inline src/feeds.ts:114]: Would it make sense that we would have an example for feeds? In describe twin example, just to check they're there or in inputs?

**Review by zoltaniotic — APPROVED** [review]
Nice one! LGTM.Just one comment. Also, I noticed if I create a twin in any example and I put a label `waterloo`, then I try to run the explorerquery it doesn't find it and the keyword in that example is waterloo. Related to [this](https://ioticlabs.atlassian.net/browse/EI-1413) ticket?

File: src/feeds.ts (hunk 1 above)
Comment (captnz) [reply src/feeds.ts:114]: Was taking me too long to try and add this so i'll create a tech debt ticket for this as ts client is not a first class citizen ok?

Comment (zoltaniotic) [reply src/feeds.ts:114]: Sure good idea

**Pull request merged**